README.md

```diff
@@ -105,16 +106,16 @@ This is useful if you want to do a computation on every icon:
 const simpleIcons = require('simple-icons');
 
 for (const iconSlug in simpleIcons) {
-    const icon = simpleIcons.Get(iconSlug);
-    // do stuff
+  const icon = simpleIcons.Get(iconSlug);
+  // do stuff
 }
 ```
 
-#### TypeScript Usage
+#### TypeScript Usage <img src="./icons/typescript.svg#gh-light-mode-only" alt="Typescript" align=left width=24 height=24><img src="./assets/readme/typescript.svg#gh-dark-mode-only" alt="Typescript" align=left width=24 height=24>
 
 Type definitions are bundled with the package.
 
-### PHP Usage
+### PHP Usage <img src="./icons/php.svg#gh-light-mode-only" alt="Php" align=left width=24 height=24><img src="./assets/readme/php.svg#gh-dark-mode-only" alt="Php" align=left width=24 height=24>
```

Comment:
Paths must be absolute or will not work in NPM not Packagist.

Reply:
Why absolute paths?

Reply:
The icons should not work in NPM or Packagist anyways. Weren't they supposed to be stripped before publishing to anything other than Github??
Since I don't have I restored them to absolute paths since you preferred it like that. I don't have strong opinions on the urls.

Reply:
The action strips the dark URLs, not absolutify also the white ones.